src/server/mongodb/actions/User.js

```diff
@@ -28,13 +28,25 @@ async function deleteUser(id) {
   return res;
 }
 async function createRootUser() {
-  return await User.create({
-    email: "root@boggers.com",
-    password: await bcrypt.hash("root", 10),
-    firstName: "boggers",
-    lastName: "boggers",
-    access: 2,
-  });
+  try {
+    const user = await User.create({
+      email: "root@boggers.com",
+      password: await bcrypt.hash("root", 10),
+      firstName: "boggers",
+      lastName: "boggers",
+      access: 2,
+      emailVerified: true,
+    });
+
+    return user;
+  } catch(error) {
+    if (error.code === 11000) {
+      throw new Error("Duplicate email")
```

Comment:
Rather than throwing an error, return a JSON so the user can read the error. See how createRootUser is used in `src/pages/api/scripts.js`